pkgs/racket-doc/scribblings/foreign/define.scrbl

```diff
@@ -97,3 +105,33 @@ with @racket[#:make-fail] or @racket[#:default-make-fail] in
 Equivalent to @racket[(provide (protect-out provide-spec ...))]. The
 @racket[provide-protected] identifier is useful with
 @racket[#:provide] in @racket[define-ffi-definer].}
+
+@section{FFI Symbol Conventions}
```

Comment:
`Symbol` should be `Identifier` (or vice-versa, as long as everything is consistent)